awx/ui_next/src/screens/Inventory/InventoryHosts/InventoryHosts.jsx

```diff
@@ -1,28 +1,22 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 
-import Host from '../../Host/Host';
+import InventoryHost from '../InventoryHost';
+import InventoryHostAdd from '../InventoryHostAdd';
 import InventoryHostList from './InventoryHostList';
-import HostAdd from '../../Host/HostAdd';
 
 function InventoryHosts({ setBreadcrumb, inventory }) {
   return (
     <Switch>
       <Route key="host-add" path="/inventories/inventory/:id/hosts/add">
-        <HostAdd />
+        <InventoryHostAdd inventory={inventory} />
+      </Route>
+      <Route key="host" path="/inventories/inventory/:id/hosts/:hostId">
+        <InventoryHost setBreadcrumb={setBreadcrumb} inventory={inventory} />
+      </Route>
+      <Route key="host-list" path="/inventories/inventory/:id/hosts">
+        <InventoryHostList />
```

Comment:
We probably want to add a Not Found route to catch erroneous urls.

Reply:
It looks like we have a Route that catches erroneous urls in the Inventories.jsx file and InventoryHost.jsx file. What url would you expect to catch in this component?

If a user navigates to `/inventories/inventory/1/hosts/1/foobar`, then the InventoryHost.jsx file should catch the route. 

If a user navigates to `/inventories/inventory/1/foobar`, then the Inventory.jsx file should catch the route.

Reply:
Ahh that's true.  Sounds good to me.